tests/functional/object/rgw/test_rgw_http_endpoint.py

```
@@ -318,31 +340,127 @@ def test_rgw_disable_http_endpoint(self):
+            params = '{"spec":{"managedResources":{"cephObjectStores":{"disableHttp": null}}}}'
+            try:
+                self.storage_cluster_obj.patch(
+                    resource_name=self.storage_cluster_name,
+                    params=params,
+                    format_type="merge",
+                )
+                log.info("disableHttp=null patch applied")
+            except CommandFailed as e:
+                log.warning(
+                    f"Setting disableHttp=null raised an error (expected for boolean CRD "
+                    f"field): {e}"
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)